src/util.h

```diff
@@ -51,6 +51,13 @@ extern bool fLogIPs;
 extern volatile bool fReopenDebugLog;
 extern CTranslationInterface translationInterface;
 
+extern const char * const BITCOIN_CONF_FILENAME;
+extern const char * const BITCOIN_PID_FILENAME;
+
+static const bool DEFAULT_SELFSIGNED_ROOTCERTS = false;
+static const bool DEFAULT_CHOOSE_DATADIR = false;
```

Comment:
For reference, @laanwj you meant `DEFAULT_SPLASHSCREEN` (and `DEFAULT_CHOOSE_DATADIR`?)?